lib/v2/alistapart/topic.js

```diff
@@ -0,0 +1,32 @@
+const { getData, getList } = require('./utils');
+
+module.exports = async (ctx) => {
+    const baseUrl = 'https://alistapart.com';
+    const searchRoute = '/wp-json/wp/v2/categories?search=';
```

Comment:
`search` parameter will return multiple results while the `slug` parameter will return the exact matched result.